util/nrt_diff/nrtDiff.py

```diff
@@ -1265,17 +1265,20 @@ def XMLCloseEnough (path_left, path_right):
 def DTWDistance(left, right) :
     n = len(left)
     m = len(right)
-    DTW = [[0 for j in range(m+1)] for i in range(n+1)]
-
-    for i in range(1, n+1):
-        for j in range (1, m+1):
-            DTW[i][j] = 999999
+    if n == m:
```

Comment:
why check the size if you check equality just after ?

Reply:
I do not know what python does if comparing two different sizes lists, but anyway, if sizes are different, we need to go throught DTW

Reply:
comparing lists compare also the size in python